File: CS/GridProject/ExtendedGridView/ExtendedGridControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;


namespace ExtendedGridViewSpace
{
    [System.ComponentModel.DesignerCategory("")]
    public class ExtendedGridControl : GridControl
    {
        protected override void RegisterAvailableViewsCore(InfoCollection collection)
        {
            base.RegisterAvailableViewsCore(collection);
            collection.Add(new ExtendedGridViewInfoRegistrator());
        }
    }
}

File: VB/GridProject/ExtendedGridView/ExtendedGridControl.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator


Namespace ExtendedGridViewSpace
	<System.ComponentModel.DesignerCategory("")>
	Public Class ExtendedGridControl
		Inherits GridControl

		Protected Overrides Sub RegisterAvailableViewsCore(ByVal collection As InfoCollection)
			MyBase.RegisterAvailableViewsCore(collection)
			collection.Add(New ExtendedGridViewInfoRegistrator())
		End Sub
	End Class
End Namespace
